guides/common/modules/proc_installing-puppet-agent-during-host-provisioning.adoc

```diff
@@ -49,11 +53,13 @@ ifndef::katello,orcharhino,satellite[]
 +
 Select the *boolean* type and set the value to `true`.
 endif::[]
-ifdef::katello,orcharhino,satellite[]
-. Enable the Puppet agent using a host parameter in global parameters, a host group, or for a single host.
-Add a host parameter named `enable-puppet7`, select the *boolean* type, and set the value to `true`.
-endif::[]
+. Set configuration for the Puppet agent.
+* If you use a non-integrated Puppet setup, set the following host parameters in global parameters or a host group:
+** Add a host parameter named `puppet_server`, select the *string* type, and set the value to the hostname of your Puppet server, such as `puppet.example.com`.
+** Optional: Add a host parameter named `puppet_ca_server`, select the *string* type, and set the value to the hostname of your Puppet CA server, such as `puppet-ca.example.com`.
+If `puppet_ca_server` is not set, the Puppet agent will use the same server as `puppet_server`.
+** Optional: Add a host parameter named `puppet_environment`, select the *string* type, and set the value to the Puppet environment you want the host to use.
+* If you use an integrated Puppet setup, ensure that you select a Puppet {SmartProxy}, Puppet CA {SmartProxy}, and Puppet environment when you create your host.
```

Comment:
I think it would be better if we keep this line first before non-integrated as this guide is specific to puppet integration of foreman